config/7.2.0/securebanking/ig/config/prod/config/config.json

```diff
@@ -422,6 +422,16 @@
       "config": {
         "validKeyUse": "tls"
       }
+    },
+    {
+      "name": "CompactSerializationJwsSigner-RSASSA-PSS",
+      "type": "com.forgerock.sapi.gateway.jws.signer.CompactSerializationJwsSigner",
```

Comment:
Add alias to SecureApiGatewayClassAliasResolver and use the short name in config.

Reply:
done